### ISSUE TITLE
Segmentation Fault Can't Run mpv Terminal

### ISSUE THREAD

**Issue by SirCartierr** [mpv-player/mpv#12910]
### Important Information

Provide following Information:
- mpv 0.36.0 Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
libplacebo version: v6.338.1
FFmpeg version: 6.0-9
FFmpeg library versions:
   libavutil       58.2.100
   libavcodec      60.3.100
   libavformat     60.3.100
   libswscale      7.1.100
   libavfilter     9.3.100
   libswresample   4.10.100

- Linux Distribution and Version: Kali Linux release 2023.3

- Source of the mpv binary: https://github.com/mpv-player/mpv.git

- GPU driver and version: Intel 8086 
<details><summary>Driver Details</summary>
<p>

```shell
>>sudo lshw -numeric -C display
  *-display                 
       description: VGA compatible controller
       product: Iris Plus Graphics G1 (Ice Lake) [8086:8A56]
       vendor: Intel Corporation [8086]
       physical id: 2
       bus info: pci@0000:00:02.0
       logical name: /dev/fb0
       version: 07
       width: 64 bits
       clock: 33MHz
       capabilities: pciexpress msi pm vga_controller bus_master cap_list rom fb
       configuration: depth=32 driver=i915 latency=0 mode=1920x1080 resolution=1920,1080 visual=truecolor xres=1920 yres=1080
       resources: iomemory:600-5ff iomemory:400-3ff irq:139 memory:6000000000-6000ffffff memory:4000000000-400fffffff ioport:3000(size=64) memory:c0000-dffff

``` 

</p>
</details> 

- Preview of Issue: 
<details><summary>Example of Output From Terminal</summary>
<p>

```shell
>>mpv -v test.mkv
[cplayer] Command line options: '-v' 'test.mkv'
[cplayer] mpv 0.36.0 Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
[cplayer]  built on UNKNOWN
[cplayer] libplacebo version: v6.338.1
[cplayer] FFmpeg version: 6.0-9
[cplayer] FFmpeg library versions:
[cplayer]    libavutil       58.2.100
[cplayer]    libavcodec      60.3.100
[cplayer]    libavformat     60.3.100
[cplayer]    libswscale      7.1.100
[cplayer]    libavfilter     9.3.100
[cplayer]    libswresample   4.10.100
[cplayer] 
[cplayer] Configuration: -Dpython.bytecompile=-1 -Dlibmpv=true -Dbuild-date=false -Dcdda=enabled -Ddvdnav=enabled -Dsdl2=enabled -Dsndio=disabled -Ddvbin=enabled -Dprefix=/usr -Dlibdir=lib/x86_64-linux-gnu -Dlocalstatedir=/var -Dsysconfdir=/etc -Dbuildtype=plain -Dwrap_mode=nodownload
[cplayer] List of enabled features: alsa av-channel-layout avif_muxer caca cdda cplugins cuda-hwaccel cuda-interop dmabuf-interop-gl dmabuf-interop-pl dmabuf-wayland drm drm-is-kms dvbin dvdnav egl egl-drm egl-helpers egl-x11 ffmpeg ffnvcodec gbm gl gl-wayland glibc-thread-name glob glob-posix gpl iconv jack javascript jpeg jpegxl lcms2 libarchive libass libavdevice libbluray libdl libm libplacebo libplacebo-next librt linux-fstatfs lua52 manpage-build memfd_create noexecstack pipewire posix posix_shm pulse rubberband rubberband-3 sdl2 sdl2-audio sdl2-gamepad sdl2-video sixel spirv-cross stdatomic threads uchardet vaapi vaapi-drm vaapi-egl vaapi-libplacebo vaapi-wayland vaapi-x-egl vaapi-x11 vdpau vector vk_khr_display vt.h vulkan wayland wayland_protocols_1_27 wayland_protocols_1_31 wayland_protocols_1_32 x11 xv zimg zimg-st428 zlib
[cplayer] Reading config file /etc/mpv/encoding-profiles.conf
[ifo_dvdnav] Opening /etc/mpv/encoding-profiles.conf
[bdmv/bluray] Opening /etc/mpv/encoding-profiles.conf
[file] Opening /etc/mpv/encoding-profiles.conf
[cplayer] Applying profile 'default'...
[cplayer] Setting option 'v' = '' (flags = 8)
[cplayer] Waiting for scripts...
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/visibility="auto" -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/margins={"l":0,"b":0,"r":0,"t":0} -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/margins={"l":0,"b":0,"r":0,"t":0} -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/margins={"l":0,"b":0,"r":0,"t":0} -> 1
[osd/libass] libass API version: 0x1701000
[osd/libass] libass source: tarball: 0.17.1
[osd/libass] Shaper: FriBidi 1.0.8 (SIMPLE) HarfBuzz-ng 8.0.1 (COMPLEX)
[osd/libass] Setting up fonts...
[osd/libass] Using font provider fontconfig
[osd/libass] Done.
[cplayer] Done loading scripts.
[cplayer] Running hook: ytdl_hook/on_load
[ytdl_hook] ytdl:// hook 
[ytdl_hook] not a ytdl:// url 
[osd/libass] libass API version: 0x1701000
[osd/libass] libass source: tarball: 0.17.1
[osd/libass] Shaper: FriBidi 1.0.8 (SIMPLE) HarfBuzz-ng 8.0.1 (COMPLEX)
[osd/libass] Setting up fonts...
[ifo_dvdnav] Opening test.mkv
[bdmv/bluray] Opening test.mkv
[file] Opening test.mkv
[demux] Trying demuxers for level=normal.
[mkv] Seeking to 18637403 to read header element 0x1c53bb6b.
[file] stream level seek from 131072 to 18637403
[mkv] Parsing cues...
[mkv] Discarding potentially broken or useless index.
[mkv] Seeking to 18637445 to read header element 0x1254c367.
[file] stream level seek from 18638099 to 5731
[mkv] All headers are parsed!
[demux] Detected file format: Matroska
[cplayer] Opening done: test.mkv
[osd/libass] Using font provider fontconfig
[osd/libass] Done.
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/margins={"l":0,"b":0,"r":0,"t":0} -> 1
[find_files] Loading external files in .
[cplayer] Running hook: ytdl_hook/on_preloaded
[mkv] select track 0
[mkv] select track 1
[cplayer]  (+) Video --vid=1 (*) (hevc 3840x2072 24.000fps)
[cplayer]  (+) Audio --aid=1 --alang=ita (*) (aac 2ch 48000Hz)
[vo/gpu] Probing for best GPU context.
[vo/gpu/opengl] Initializing GPU context 'wayland'
[vo/gpu/opengl] Initializing GPU context 'x11egl'
[vo/gpu/x11] X11 opening display: :0.0
[vo/gpu/x11] Display 0 (eDP-1): [0, 0, 1920, 1080] @ 60.007752 FPS
[vo/gpu/x11] Current display FPS: 60.007752
[vo/gpu/opengl] EGL_VERSION=1.5
[vo/gpu/opengl] EGL_VENDOR=Mesa Project
[vo/gpu/opengl] EGL_CLIENT_APIS=OpenGL OpenGL_ES 
[vo/gpu/opengl] Trying to create Desktop OpenGL context.
[vo/gpu/opengl] Choosing visual EGL config 0x15, visual ID 0x21
[vo/gpu/x11] The XPresent extension was found.
[vo/gpu/x11] XPresent enabled.
[vo/gpu/opengl] GL_VERSION='4.6 (Core Profile) Mesa 23.2.1-1'
[vo/gpu/opengl] Detected desktop OpenGL 4.6.
[vo/gpu/opengl] GL_VENDOR='Intel'
[vo/gpu/opengl] GL_RENDERER='Mesa Intel(R) UHD Graphics (ICL GT1)'
[vo/gpu/opengl] GL_SHADING_LANGUAGE_VERSION='4.60'
[vo/gpu/opengl] Loaded extension GL_KHR_debug.
[vo/gpu] Testing FBO format rgba16f
[vo/gpu] Using FBO format rgba16f.
[vo/gpu] No advanced processing required. Enabling dumb mode.
[vo/gpu] Assuming 60.007752 FPS for display sync.
[vd] Container reported FPS: 24.000000
[vd] Codec list:
[vd]     hevc - HEVC (High Efficiency Video Coding)
[vd]     hevc_qsv (hevc) - HEVC video (Intel Quick Sync Video acceleration)
[vd]     hevc_v4l2m2m (hevc) - V4L2 mem2mem HEVC decoder wrapper
[vd]     hevc_cuvid (hevc) - Nvidia CUVID HEVC decoder
[vd] Opening decoder hevc
[vd] No hardware decoding requested.
[vd] Using software decoding.
[vd] Detected 8 logical cores.
[vd] Requesting 9 threads for decoding.
[vd] Selected codec: hevc (HEVC (High Efficiency Video Coding))
[vf] User filter list:
[vf]   (empty)
[ad] Codec list:
[ad]     aac - AAC (Advanced Audio Coding)
[ad]     aac_fixed (aac) - AAC (Advanced Audio Coding)
[ad] Opening decoder aac
[ad] Requesting 1 threads for decoding.
[ad] Selected codec: aac (AAC (Advanced Audio Coding))
[af] User filter list:
[af]   (empty)
[cplayer] Starting playback...
[af] [in] 48000Hz stereo 2ch floatp
[af] [userspeed] 48000Hz stereo 2ch floatp
[af] [userspeed] (disabled)
[af] [convert] 48000Hz stereo 2ch floatp
[vo/gpu] DR path suspected slow/uncached, disabling.
[vd] DR failed - disabling.
[ao] Trying audio driver 'pipewire'
[ao/pipewire] requested format: 48000 Hz, stereo channels, floatp
[ao/pipewire] Headers version: 0.3.84
[ao/pipewire] Library version: 0.3.84
[ao/pipewire] Core user: sim
[ao/pipewire] Core host: KaliPC
[ao/pipewire] Core version: 0.3.84
[ao/pipewire] Core name: pipewire-0
[ao/pipewire] using soft-buffer of 9600 samples.
[cplayer] AO: [pipewire] 48000Hz stereo 2ch floatp
[cplayer] AO: Description: PipeWire audio output
[af] [convert] (disabled)
[af] [out] 48000Hz stereo 2ch floatp
[vd] Using software decoding.
[vd] Decoder format: 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/auto SP=4.926108 CL=mpeg2/4/h264
[vd] Using container aspect ratio.
[vf] [in] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vf] [userdeint] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vf] [userdeint] (disabled)
[vf] [autorotate] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vf] [autorotate] (disabled)
[vf] [convert] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vf] [convert] (disabled)
[vf] [out] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[cplayer] VO: [gpu] 3840x2072 yuv420p10
[cplayer] VO: Description: Shader-based GPU Renderer
[vo/gpu] reconfig to 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vo/gpu/x11] not waiting for MapNotify
[vo/gpu] Resize: 1910x1015
[vo/gpu] Window size: 1910x1015 (Borders: l=0 t=0 r=0 b=0)
[vo/gpu] Video source: 3840x2072 (1:1)
[vo/gpu] Video display: (0, 0) 3840x2072 -> (14, 0) 1881x1015
[vo/gpu] Video scale: 0.489844/0.489865
[vo/gpu] OSD borders: l=14 t=0 r=15 b=0
[vo/gpu] Video borders: l=14 t=0 r=15 b=0
[vo/gpu] Reported display depth: 8
[vo/gpu] Texture for plane 0: 3840x2072
[vo/gpu] Texture for plane 1: 1920x1036
[vo/gpu] Texture for plane 2: 1920x1036
[vo/gpu] Testing FBO format rgba16f
[vo/gpu] Using FBO format rgba16f.
[vo/gpu] No advanced processing required. Enabling dumb mode.
[vo/gpu] Resize: 1910x1015
[vo/gpu] Window size: 1910x1015 (Borders: l=0 t=0 r=0 b=0)
[vo/gpu] Video source: 3840x2072 (1:1)
[vo/gpu] Video display: (0, 0) 3840x2072 -> (14, 0) 1881x1015
[vo/gpu] Video scale: 0.489844/0.489865
[vo/gpu] OSD borders: l=14 t=0 r=15 b=0
[vo/gpu] Video borders: l=14 t=0 r=15 b=0
[vo/gpu] Reported display depth: 8
[1]    123698 segmentation fault  mpv -v test.mkv

``` 

</p>
</details> 

### Reproduction steps

Try to reproduce your issue with --no-config first. If it isn't reproducible
with --no-config try to first find out which option or script causes your issue.

<details><summary>no-config test</summary>
<p>

```shell
>>mpv -v --no-config test.mkv 
[cplayer] Command line options: '-v' '--no-config' 'test.mkv'
[cplayer] mpv 0.36.0 Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
[cplayer]  built on UNKNOWN
[cplayer] libplacebo version: v6.338.1
[cplayer] FFmpeg version: 6.0-9
[cplayer] FFmpeg library versions:
[cplayer]    libavutil       58.2.100
[cplayer]    libavcodec      60.3.100
[cplayer]    libavformat     60.3.100
[cplayer]    libswscale      7.1.100
[cplayer]    libavfilter     9.3.100
[cplayer]    libswresample   4.10.100
[cplayer] 
[cplayer] Configuration: -Dpython.bytecompile=-1 -Dlibmpv=true -Dbuild-date=false -Dcdda=enabled -Ddvdnav=enabled -Dsdl2=enabled -Dsndio=disabled -Ddvbin=enabled -Dprefix=/usr -Dlibdir=lib/x86_64-linux-gnu -Dlocalstatedir=/var -Dsysconfdir=/etc -Dbuildtype=plain -Dwrap_mode=nodownload
[cplayer] List of enabled features: alsa av-channel-layout avif_muxer caca cdda cplugins cuda-hwaccel cuda-interop dmabuf-interop-gl dmabuf-interop-pl dmabuf-wayland drm drm-is-kms dvbin dvdnav egl egl-drm egl-helpers egl-x11 ffmpeg ffnvcodec gbm gl gl-wayland glibc-thread-name glob glob-posix gpl iconv jack javascript jpeg jpegxl lcms2 libarchive libass libavdevice libbluray libdl libm libplacebo libplacebo-next librt linux-fstatfs lua52 manpage-build memfd_create noexecstack pipewire posix posix_shm pulse rubberband rubberband-3 sdl2 sdl2-audio sdl2-gamepad sdl2-video sixel spirv-cross stdatomic threads uchardet vaapi vaapi-drm vaapi-egl vaapi-libplacebo vaapi-wayland vaapi-x-egl vaapi-x11 vdpau vector vk_khr_display vt.h vulkan wayland wayland_protocols_1_27 wayland_protocols_1_31 wayland_protocols_1_32 x11 xv zimg zimg-st428 zlib
[cplayer] Setting option 'v' = '' (flags = 8)
[cplayer] Setting option 'config' = 'no' (flags = 8)
[cplayer] Waiting for scripts...
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/visibility="auto" -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/margins={"b":0,"r":0,"t":0,"l":0} -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/margins={"b":0,"r":0,"t":0,"l":0} -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: user-data/osc/margins={"b":0,"r":0,"t":0,"l":0} -> 1
[osd/libass] libass API version: 0x1701000
[osd/libass] libass source: tarball: 0.17.1
[osd/libass] Shaper: FriBidi 1.0.8 (SIMPLE) HarfBuzz-ng 8.0.1 (COMPLEX)
[osd/libass] Setting up fonts...
[osd/libass] Using font provider fontconfig
[osd/libass] Done.
[osd/libass] libass API version: 0x1701000
[osd/libass] libass source: tarball: 0.17.1
[osd/libass] Shaper: FriBidi 1.0.8 (SIMPLE) HarfBuzz-ng 8.0.1 (COMPLEX)
[osd/libass] Setting up fonts...
[osd/libass] Using font provider fontconfig
[osd/libass] Done.
[cplayer] Done loading scripts.
[cplayer] Running hook: ytdl_hook/on_load
[ytdl_hook] ytdl:// hook 
[ytdl_hook] not a ytdl:// url 
[ifo_dvdnav] Opening test.mkv
[bdmv/bluray] Opening test.mkv
[file] Opening test.mkv
[demux] Trying demuxers for level=normal.
[mkv] Seeking to 18637403 to read header element 0x1c53bb6b.
[file] stream level seek from 131072 to 18637403
[mkv] Parsing cues...
[mkv] Discarding potentially broken or useless index.
[mkv] Seeking to 18637445 to read header element 0x1254c367.
[file] stream level seek from 18638099 to 5731
[mkv] All headers are parsed!
[demux] Detected file format: Matroska
[cplayer] Opening done: test.mkv
[find_files] Loading external files in .
[cplayer] Running hook: ytdl_hook/on_preloaded
[mkv] select track 0
[mkv] select track 1
[cplayer]  (+) Video --vid=1 (*) (hevc 3840x2072 24.000fps)
[cplayer]  (+) Audio --aid=1 --alang=ita (*) (aac 2ch 48000Hz)
[vo/gpu] Probing for best GPU context.
[vo/gpu/opengl] Initializing GPU context 'wayland'
[vo/gpu/opengl] Initializing GPU context 'x11egl'
[vo/gpu/x11] X11 opening display: :0.0
[vo/gpu/x11] Display 0 (eDP-1): [0, 0, 1920, 1080] @ 60.007752 FPS
[vo/gpu/x11] Current display FPS: 60.007752
[vo/gpu/opengl] EGL_VERSION=1.5
[vo/gpu/opengl] EGL_VENDOR=Mesa Project
[vo/gpu/opengl] EGL_CLIENT_APIS=OpenGL OpenGL_ES 
[vo/gpu/opengl] Trying to create Desktop OpenGL context.
[vo/gpu/opengl] Choosing visual EGL config 0x15, visual ID 0x21
[vo/gpu/x11] The XPresent extension was found.
[vo/gpu/x11] XPresent enabled.
[vo/gpu/opengl] GL_VERSION='4.6 (Core Profile) Mesa 23.2.1-1'
[vo/gpu/opengl] Detected desktop OpenGL 4.6.
[vo/gpu/opengl] GL_VENDOR='Intel'
[vo/gpu/opengl] GL_RENDERER='Mesa Intel(R) UHD Graphics (ICL GT1)'
[vo/gpu/opengl] GL_SHADING_LANGUAGE_VERSION='4.60'
[vo/gpu/opengl] Loaded extension GL_KHR_debug.
[vo/gpu] Testing FBO format rgba16f
[vo/gpu] Using FBO format rgba16f.
[vo/gpu] No advanced processing required. Enabling dumb mode.
[vo/gpu] Assuming 60.007752 FPS for display sync.
[vd] Container reported FPS: 24.000000
[vd] Codec list:
[vd]     hevc - HEVC (High Efficiency Video Coding)
[vd]     hevc_qsv (hevc) - HEVC video (Intel Quick Sync Video acceleration)
[vd]     hevc_v4l2m2m (hevc) - V4L2 mem2mem HEVC decoder wrapper
[vd]     hevc_cuvid (hevc) - Nvidia CUVID HEVC decoder
[vd] Opening decoder hevc
[vd] No hardware decoding requested.
[vd] Using software decoding.
[vd] Detected 8 logical cores.
[vd] Requesting 9 threads for decoding.
[vd] Selected codec: hevc (HEVC (High Efficiency Video Coding))
[vf] User filter list:
[vf]   (empty)
[ad] Codec list:
[ad]     aac - AAC (Advanced Audio Coding)
[ad]     aac_fixed (aac) - AAC (Advanced Audio Coding)
[ad] Opening decoder aac
[ad] Requesting 1 threads for decoding.
[ad] Selected codec: aac (AAC (Advanced Audio Coding))
[af] User filter list:
[af]   (empty)
[cplayer] Starting playback...
[vo/gpu] DR path suspected slow/uncached, disabling.
[vd] DR failed - disabling.
[vd] Using software decoding.
[vd] Decoder format: 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/auto SP=4.926108 CL=mpeg2/4/h264
[vd] Using container aspect ratio.
[vf] [in] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vf] [userdeint] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vf] [userdeint] (disabled)
[vf] [autorotate] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vf] [autorotate] (disabled)
[vf] [convert] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vf] [convert] (disabled)
[vf] [out] 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[af] [in] 48000Hz stereo 2ch floatp
[af] [userspeed] 48000Hz stereo 2ch floatp
[af] [userspeed] (disabled)
[af] [convert] 48000Hz stereo 2ch floatp
[ao] Trying audio driver 'pipewire'
[ao/pipewire] requested format: 48000 Hz, stereo channels, floatp
[ao/pipewire] Headers version: 0.3.84
[ao/pipewire] Library version: 0.3.84
[ao/pipewire] Core user: sim
[ao/pipewire] Core host: KaliPC
[ao/pipewire] Core version: 0.3.84
[ao/pipewire] Core name: pipewire-0
[ao/pipewire] using soft-buffer of 9600 samples.
[cplayer] AO: [pipewire] 48000Hz stereo 2ch floatp
[cplayer] AO: Description: PipeWire audio output
[af] [convert] (disabled)
[af] [out] 48000Hz stereo 2ch floatp
[cplayer] VO: [gpu] 3840x2072 yuv420p10
[cplayer] VO: Description: Shader-based GPU Renderer
[vo/gpu] reconfig to 3840x2072 yuv420p10 bt.2020-ncl/bt.2020/pq/limited/display SP=4.926108 CL=mpeg2/4/h264
[vo/gpu/x11] not waiting for MapNotify
[vo/gpu] Resize: 1910x1015
[vo/gpu] Window size: 1910x1015 (Borders: l=0 t=0 r=0 b=0)
[vo/gpu] Video source: 3840x2072 (1:1)
[vo/gpu] Video display: (0, 0) 3840x2072 -> (14, 0) 1881x1015
[vo/gpu] Video scale: 0.489844/0.489865
[vo/gpu] OSD borders: l=14 t=0 r=15 b=0
[vo/gpu] Video borders: l=14 t=0 r=15 b=0
[vo/gpu] Reported display depth: 8
[vo/gpu] Texture for plane 0: 3840x2072
[vo/gpu] Texture for plane 1: 1920x1036
[vo/gpu] Texture for plane 2: 1920x1036
[vo/gpu] Testing FBO format rgba16f
[vo/gpu] Using FBO format rgba16f.
[vo/gpu] No advanced processing required. Enabling dumb mode.
[vo/gpu] Resize: 1910x1015
[vo/gpu] Window size: 1910x1015 (Borders: l=0 t=0 r=0 b=0)
[vo/gpu] Video source: 3840x2072 (1:1)
[vo/gpu] Video display: (0, 0) 3840x2072 -> (14, 0) 1881x1015
[vo/gpu] Video scale: 0.489844/0.489865
[vo/gpu] OSD borders: l=14 t=0 r=15 b=0
[vo/gpu] Video borders: l=14 t=0 r=15 b=0
[vo/gpu] Reported display depth: 8
[1]    124391 segmentation fault  mpv -v --no-config test.mkv

``` 

</p>
</details> 

Describe the reproduction steps as precise as possible. It's very likely that
the bug you experience wasn't reproduced by the developer because the workflow
differs from your own.
- I think I more than likely did something to my mpv config file a few days ago because mpv worked fine for the first few days I had it (I only have had mpv for ~1week).

### Expected behavior

For mpv to run a video file from my terminal.

### Actual behavior

Segmentation fault

### Log file

Make a log file made with -v -v or --log-file=output.txt, paste it to
https://0x0.st/ or attach it to the github issue, and replace this text with a
link to it.

<details><summary>gdb</summary>
<p>

```shell
>>gdb mpv
GNU gdb (Debian 13.2-1) 13.2
Copyright (C) 2023 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from mpv...
(No debugging symbols found in mpv)
(gdb) run test.mkv
Starting program: /usr/bin/mpv test.mkv
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7fffe45ff6c0 (LWP 118748)]
[New Thread 0x7fffe3dfe6c0 (LWP 118749)]
[New Thread 0x7fffd35fd6c0 (LWP 118750)]
[New Thread 0x7fffd2dfc6c0 (LWP 118751)]
[New Thread 0x7fffc25fb6c0 (LWP 118752)]
[New Thread 0x7fffb9dfa6c0 (LWP 118753)]
[New Thread 0x7fffb95f96c0 (LWP 118754)]
[New Thread 0x7fffa8df86c0 (LWP 118755)]
[New Thread 0x7fffa85f76c0 (LWP 118756)]
[New Thread 0x7fffa7df66c0 (LWP 118757)]
[New Thread 0x7fffa75f56c0 (LWP 118758)]
[New Thread 0x7fffa6df46c0 (LWP 118759)]
[New Thread 0x7fffa65f36c0 (LWP 118760)]
[Thread 0x7fffa65f36c0 (LWP 118760) exited]
[New Thread 0x7fffa65f36c0 (LWP 118761)]
[mkv] Discarding potentially broken or useless index.
[New Thread 0x7fffa65f36c0 (LWP 118762)]
[Thread 0x7fffa65f36c0 (LWP 118761) exited]
[New Thread 0x7fffa57666c0 (LWP 118763)]
 (+) Video --vid=1 (*) (hevc 3840x2072 24.000fps)
 (+) Audio --aid=1 --alang=ita (*) (aac 2ch 48000Hz)
[New Thread 0x7fffa4f5e6c0 (LWP 118764)]
[New Thread 0x7fffa4f5e6c0 (LWP 118765)]
[Thread 0x7fffa4f5e6c0 (LWP 118764) exited]
[New Thread 0x7fff7cdff6c0 (LWP 118766)]
[New Thread 0x7fff77fff6c0 (LWP 118767)]
[New Thread 0x7fff777fe6c0 (LWP 118768)]
[New Thread 0x7fff76ffd6c0 (LWP 118769)]
[New Thread 0x7fff767fc6c0 (LWP 118770)]
[New Thread 0x7fff75ffb6c0 (LWP 118771)]
[New Thread 0x7fff757fa6c0 (LWP 118772)]
[New Thread 0x7fff74ff96c0 (LWP 118773)]
[New Thread 0x7fff57fff6c0 (LWP 118774)]
[New Thread 0x7fff577fe6c0 (LWP 118775)]
[New Thread 0x7fff56ffd6c0 (LWP 118776)]
[New Thread 0x7fff567fc6c0 (LWP 118777)]
[New Thread 0x7fff55ffb6c0 (LWP 118778)]
[New Thread 0x7fff557fa6c0 (LWP 118779)]
[New Thread 0x7fff54ff96c0 (LWP 118780)]
[New Thread 0x7fff4ffff6c0 (LWP 118781)]
[New Thread 0x7fff4f7fe6c0 (LWP 118782)]
[New Thread 0x7fff4effd6c0 (LWP 118783)]
[New Thread 0x7fff1f28a6c0 (LWP 118784)]
[New Thread 0x7fff1ea896c0 (LWP 118785)]
[New Thread 0x7fff1e2886c0 (LWP 118786)]
AO: [pipewire] 48000Hz stereo 2ch floatp
VO: [gpu] 3840x2072 yuv420p10

Thread 19 "mpv/vo" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7fffa4f5e6c0 (LWP 118765)]
0x0000000000000000 in ?? ()
(gdb) 

``` 

</p>
</details> 

The issue will be closed for ignoring the issue template.

### Sample files

Sample files needed to reproduce this issue can be uploaded to https://0x0.st/
or similar sites. (Only needed if the issue cannot be reproduced without it.)
Do not use garbage like "cloud storage", especially not Google Drive.

**Comment by llyyr** [comment]
That backtrace is not helpful, please follow https://wiki.debian.org/HowToGetABacktrace or https://wiki.archlinux.org/title/Debugging/Getting_traces#Getting_the_trace

**Comment by SirCartierr** [comment]
> That backtrace is not helpful, please follow https://wiki.debian.org/HowToGetABacktrace or https://wiki.archlinux.org/title/Debugging/Getting_traces#Getting_the_trace

Will do that now. Thanks for the links I'll update my post after.

**Comment by SirCartierr** [comment]
> That backtrace is not helpful, please follow https://wiki.debian.org/HowToGetABacktrace or https://wiki.archlinux.org/title/Debugging/Getting_traces#Getting_the_trace

I am getting this error when trying to setup an output for a log file. 

```shell
$ mpv --log-file=output.txt "/home/sim/test.mkv"
Failed to open log file 'output.txt'
[mkv] Discarding potentially broken or useless index.
 (+) Video --vid=1 (*) (hevc 3840x2072 24.000fps)
 (+) Audio --aid=1 --alang=ita (*) (aac 2ch 48000Hz)
AO: [pipewire] 48000Hz stereo 2ch floatp
VO: [gpu] 3840x2072 yuv420p10
Segmentation fault


``` 

I am also getting this response when I try to use gdp to get backlogs on mpv. Sorry I am clearly very new to linux, but really appreciate guidance a lot!

```shell
(gdb) run mpv
Starting program:  mpv
No executable file specified.
Use the "file" or "exec-file" command.

```

**Comment by kasper93** [comment]
Can you try latest HEAD version?

**Comment by mia-0** [comment]
Possibly related: https://gitlab.freedesktop.org/mesa/mesa/-/issues/2718

**Comment by sfan5** [comment]
@SirCartierr try `gdb --batch -iex 'set confirm off' -ex r -ex bt --args mpv -v "/home/sim/test.mkv"`

**Comment by ShiroYasha139** [comment]
Dunno if I should create separate issue but I'm also having the same issue.
I'm on EndeavourOS 64bit, downloaded mpv from AUR on amdgpu

[gdbtrace.txt](https://github.com/mpv-player/mpv/files/13456258/gdbtrace.txt)
[log_output.txt](https://github.com/mpv-player/mpv/files/13456259/log_output.txt)

**Comment by sfan5** [comment]
Both of you should update to 0.37.0 and file another issue if the bug is still present.

**Comment by llyyr** [comment]
> Dunno if I should create separate issue but I'm also having the same issue.

It's different, and your crash is due to glslang update to 13.1.1. Downgrading it supposedly fixes it for now

**Comment by gergo-salyi** [comment]
> Dunno if I should create separate issue but I'm also having the same issue. I'm on EndeavourOS 64bit, downloaded mpv from AUR on amdgpu
> 
> [gdbtrace.txt](https://github.com/mpv-player/mpv/files/13456258/gdbtrace.txt) [log_output.txt](https://github.com/mpv-player/mpv/files/13456259/log_output.txt)
> 
> Edit: removing gpu-context=x11vk from mpv.conf worked

> It's different, and your crash is due to glslang update to 13.1.1. Downgrading it supposedly fixes it for now

@ShiroYasha139 , and for the record, that one should be solved on Arch-based distros with shaderc-2023.7 Arch Linux package being released now. Go back to the current official glslang-13.1.1 and update to get shaderc-2023.7 to have it fixed.
